lib/core/src/lib/dialogs/dialog/dialog.component.scss

```diff
@@ -271,14 +281,10 @@ $dialog-padding: 24px;
     color: var(--theme-primary-color);
 }
 
-// TODO: Update after migration to Angular 15
-/* stylelint-disable-next-line selector-class-pattern */
-.mat-dialog-actions .mat-button-base + .mat-button-base {
+#{$mat-dialog-actions} #{$mat-button-base} + #{$mat-button-base} {
```

Comment:
this can be removed as well, you can have something like `.adf-dialog-actions`, just introduce a custom class for the actions container, and you can style children easy, i.e. 

```css
.adf-dialog-actions {
  display: flex;
  column-gap: 16px
}
```

for the dialog surface can be found something as well.
if you can't access the parent dialog class - the library allows passing a custom dialog panel class when invoking a dialog

Reply:
added, but `column-gap: 8px` as we still have margin `8px`